examples/paper-self-gravitating-gas-dynamics/elixir_eulergravity_eoc_test_p4est.jl

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
Please move this file to the regular folder `examples/2d`. The `paper-...` folder only holds elixirs used in the corresponding paper.